src/ApiService/ApiService/Functions/Webhooks.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System.Net;
+
+namespace Microsoft.OneFuzz.Service.Functions;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+
+
+public class Webhooks {
+    private readonly ILogTracer _log;
+    private readonly IEndpointAuthorization _auth;
+    private readonly IOnefuzzContext _context;
+
+    public Webhooks(ILogTracer log, IEndpointAuthorization auth, IOnefuzzContext context) {
+        _log = log;
+        _auth = auth;
+        _context = context;
+    }
+
+    [Function("Webhooks")]
+    public Async.Task<HttpResponseData> Run(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "GET", "POST", "DELETE", "PATCH")] HttpRequestData req) {
+        return _auth.CallIfUser(req, r => r.Method switch {
+            "GET" => Get(r),
+            "POST" => Post(r),
+            "DELETE" => Delete(r),
+            "PATCH" => Patch(r),
+            _ => throw new InvalidOperationException("Unsupported HTTP method"),
+        });
+    }
+
+
+    private async Async.Task<HttpResponseData> Get(HttpRequestData req) {
+        var request = await RequestHandling.ParseRequest<WebhookSearch>(req);
+        if (!request.IsOk) {
+            return await _context.RequestHandling.NotOk(req, request.ErrorV, "webhook get");
+        }
+
+        if (request.OkV.WebhookId != null) {
+            _log.Info($"getting webhook: {request.OkV.WebhookId}");
+            var webhook = await _context.WebhookOperations.GetByWebhookId(request.OkV.WebhookId.Value);
+
+            if (webhook == null) {
+                return await _context.RequestHandling.NotOk(req, new Error(ErrorCode.INVALID_REQUEST, new[] { "unable to find webhook" }), "webhook get");
+            }
+
+            var response = req.CreateResponse(HttpStatusCode.OK);
+            await response.WriteAsJsonAsync(webhook);
+            return response;
```

Comment:
Feel free to ignore but you can use:
https://github.com/microsoft/onefuzz/blob/8738ef0044f8f6c7f9c79f1e377c899c04328a2d/src/ApiService/ApiService/onefuzzlib/Request.cs#L101